microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/integration/FailSafeCDIHealthBaseTest.java

```diff
@@ -153,7 +180,7 @@ public void testReadinessEndpointUp() throws Exception {
      */
     @Ignore("WFLY-12924, WFLY-12925")
     @Test
-    public void testHealthEndpointDownInMaintenace() throws Exception {
+    final public void testHealthEndpointDownInMaintenace() throws Exception {
```

Comment:
Typo: `..InMaintenace` -> `..InMaintenance`, the same applies for the following methods.